src/App.js

```diff
@@ -70,4 +69,25 @@ class App extends React.Component {
         );
     }
 }
-export default App;
+
+App.propTypes = {
+    state: PropTypes.object.isRequired,
+    actions: PropTypes.object.isRequired,
+};
+
+function mapStateToProps(state) {
+    return {
+        state,
+    };
+}
+
+function mapDispatchToProps(dispatch) {
+    return {
+        actions: bindActionCreators(appActions, dispatch),
```

Comment:
Why do you use the `actions` key? And what are the `appActions`? Please specify each one separately so it is easier to understand what happens here.